types/node/fs/promises.d.ts

```diff
@@ -960,7 +964,7 @@ declare module 'fs/promises' {
      * If `persistent` is not supplied, the default of `true` is used.
      * If `recursive` is not supplied, the default of `false` is used.
      */
-    function watch(filename: PathLike, options?: WatchOptions | BufferEncoding): AsyncIterable<string>;
+    function watch(filename: PathLike, options?: WatchOptions | BufferEncoding): AsyncIterable<FileChangeInfo>;
```

Comment:
See above.

Reply:
Was this supposed to be `FileChangeInfo<string>`?  It looks like the original signature didn't expect a `Buffer` to be returned (which may have been incorrect).